src/transformers/testing_utils.py

```diff
@@ -172,6 +172,19 @@ def require_torch(test_case):
         return test_case
 
 
+def require_torch_scatter(test_case):
+    """
+    Decorator marking a test that requires PyTorch scatter.
+
+    These tests are skipped when PyTorch scatter isn't installed.
+
```

Comment:
Could be on one line ;-)

Reply:
Let's refactor all of these at the same time, I used the same format as every other require defined in this file :)